<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проекты</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb( 181, 213, 202);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_select">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Выбор проекта</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_project">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 186, 191);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 186, 191);</string>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auth_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>620</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 186, 191);</string>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_project">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>651</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
alternate-background-color: #FFFCD6;
 gridline-color: #B0BEC5;
 border-radius: 5px;</string>
    </property>
    <column>
     <property name="text">
      <string>Заказ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Готовый</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="new_pr_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 186, 191);</string>
    </property>
    <property name="text">
     <string>Новый продукт</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
